azure-pipelines.yml

```
@@ -34,12 +34,36 @@ jobs:
   steps:
   - script: choco install -y winpcap
   - script: dotnet restore -s https://api.nuget.org/v3/index.json
-  - script: bash scripts/test.sh --filter
+  # Disable WinDivert to avoid an internal CLR error due to secure boot having been enabled
+  - script: bash scripts/test.sh --filter TestCategory!=WinDivert
+
+- job: windows_2016_vs2017_npcap
+  pool:
+    vmImage: 'VS2017-Win2016'
+  steps:
+  - pwsh: ./scripts/InstallNpcap.ps1
+    env:
+      npcap_oem_key: $(npcap_oem_key)
+  - script: dotnet restore -s https://api.nuget.org/v3/index.json
+  # Disable WinDivert to avoid an internal CLR error due to secure boot having been enabled
+  - script: bash scripts/test.sh --filter "TestCategory!=RemotePcap&TestCategory!=WinDivert"
 
 - job: windows_2019_vs2019
```

I think we should indicate in the name that this one uses winpcap
```suggestion
- job: windows_2019_vs2019_winpcap
```

---

I like it, let me fix.